script.js

```diff
@@ -151,19 +251,19 @@ function showMoneyQuestion(num) {
 function showTrueAnswers(event) {
   let target = event.target;
   let thisAnswers = document.querySelectorAll('.question-active .question__answers .answer__radio');
-  let falseAnswers = Array.from(thisAnswers).filter(answer => !correctAnswers.includes(answer.value));
-  let randomNum = function() {
-    return Math.floor(Math.random() * 3);
-  }
+  let falseAnswers = Array.from(thisAnswers).filter(answer => {
```

Comment:
Codacy has a fix for the issue: [Expected parentheses around arrow function argument.](https://app.codacy.com/gh/voronkova-ab/-game-Who-want-to-be-a-millionaire-/pullRequest?prid=7740148)
```suggestion
  let falseAnswers = Array.from(thisAnswers).filter((answer) => {
```